react/Wizard/Readme.md

```diff
@@ -12,17 +12,17 @@ import {
 } from '.'
 import Icon from 'cozy-ui/transpiled/react/Icon'
 import Field from 'cozy-ui/transpiled/react/Field'
-import Button from 'cozy-ui/transpiled/react/Button';
-import PreviousIcon from 'cozy-ui/transpiled/react/Icons/Previous' 
-import NextIcon from 'cozy-ui/transpiled/react/Icons/Next';
+import Button from 'cozy-ui/transpiled/react/Button'
+import PreviousIcon from 'cozy-ui/transpiled/react/Icons/Previous'
+import NextIcon from 'cozy-ui/transpiled/react/Icons/Next'
 
-const t = x => x;
-const isTiny = false;
-const error = true;
-const fetching = false;
-const onboarding = false;
+const t = x => x
+const isTiny = false
+const error = true
+const fetching = false
+const onboarding = false
```

Comment:
we can remove it from here too

Reply:
J'ai pas compris ?

Reply:
This variable is not used anymore. We should remove it as you removed it from the button props

Reply:
Ah tu parles que de `onboarding`  🤦 !! Je pensais que tu parlais de toutes les variables, ok !